ggml-opencl.cpp

```diff
@@ -1493,7 +1493,7 @@ static void ggml_cl_mul_mat_f32(const ggml_tensor * src0, const ggml_tensor * sr
     if (src0->backend == GGML_BACKEND_GPU) { // NOLINT
         d_X = (cl_mem) src0->data;
     } else {
-        d_X = ggml_cl_pool_malloc(sizeof(ggml_fp16_t) * x_ne, &x_size);
+        d_X = ggml_cl_pool_malloc(sizeof(float) * x_ne, &x_size);
```

Comment:
Isn't it better to choose the size based on `src0` type?

Reply:
In `ggml_cl_mul_mat()`, the function calls of both `ggml_cl_mul_mat_f32()` and `ggml_cl_mul_mat_f16()` indicate the `src0` type as f32 or f16 already.
I think it is a coding style issue, and in this commit, there is no need to modify other parts of the code for now.